src/components/toggleButton/ToggleButton.js

```diff
@@ -0,0 +1,9 @@
+import './toggle-button.scss';
+
+export const ToggleButton = (props) => {
+  const { activeState, clickHandler } = props;
+
+  return (
+    <button type="button" className={activeState ? 'active' : ''} onClick={() => clickHandler()}>Toggle Active</button>
```

Comment:
Button's state should be inside button component, it shouldn't be taken somewhere from outside